ChatIntegrationExamples/SendChatNotificationCard.xml

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<DMSScript options="272" xmlns="http://www.skyline.be/automation">
+	<Name>Send Chat Notification Card Example</Name>
+	<Description>Sends a notification using an Adaptive Card in a given groups chat or one-on-one chat in Microsoft Teams. Make sure to have a tenant set for your organization in the Admin App (https://admin.dataminer.services).</Description>
```

Comment:
Do you mean "in a given group's chat"? Note also that there's no need to capitalize "app". (This may need to be adjusted in other places as well.)

```suggestion
	<Description>Sends a notification using an Adaptive Card in a given group's chat or one-on-one chat in Microsoft Teams. Make sure to have a tenant set for your organization in the Admin app (https://admin.dataminer.services).</Description>
```